GameServerLib/Content/ContentManager.cs

```diff
@@ -217,47 +99,144 @@ public CharData GetCharData(string charName)
             {
                 return _charData[charName];
             }
+
             _charData[charName] = new CharData(_game);
             _charData[charName].Load(charName);
             return _charData[charName];
         }
 
+        public NavGrid GetNavGrid(int mapId)
+        {
+            var possibilities = new[]
+            {
+                $"LEVELS/Map{mapId}/AIPath.aimesh_ngrid",
+                $"LEVELS/map{mapId}/AIPath.aimesh_ngrid"
+            };
+            foreach (var path in possibilities)
+            {
+                if (_navGrids.ContainsKey(path))
+                {
+                    return _navGrids[path];
```

Comment:
Same here.

Reply:
I don't think I can write this one as LINQ.